src/features/ListNavigation/index.tsx

```diff
@@ -0,0 +1,2 @@
+import ListNavigation from "./ListNavigation";
+export default ListNavigation;
```

Comment:
if you get extra files in the folder? 
I suggest using aggregate export example: export * from './fileName'.